build.gradle

```diff
@@ -179,6 +183,15 @@ dependencies {
     implementation "org.glassfish:jakarta.json:2.0.1"
     implementation "org.eclipse:yasson:3.0.4"
     implementation "com.google.code.gson:gson:2.11.0"
+    // Swagger-Parser dependencies for API consistency tests
+    implementation "io.swagger.core.v3:swagger-models:${swaggerCoreVersion}"
+    implementation "io.swagger.core.v3:swagger-core:${swaggerCoreVersion}"
+    implementation "io.swagger.parser.v3:swagger-parser-core:${swaggerVersion}"
+    implementation "io.swagger.parser.v3:swagger-parser:${swaggerVersion}"
+    implementation "io.swagger.parser.v3:swagger-parser-v3:${swaggerVersion}"
+    implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVersion}"
+    implementation "com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}"
```

Comment:
jackson dependencies are forced on L205. L192-194 should be moved under configurations as well

Reply:
Thank you for your suggestion. I tried moving the Jackson dependencies under configurations with force, but it seems that doing so doesn't properly import the dependencies. The force directive only enforces specific versions if the dependencies are already present, but it doesn't add them to the project on its own. Therefore, the implementation lines in dependencies are still necessary to ensure the Jackson libraries are actually included.

Reply:
> only enforces specific versions 

And you are using the same version, right? 2.18.0. Any error you are seeing?

Reply:
Yes, I am using version 2.18.0 as specified in the force directive. The error I'm encountering is:

`java.lang.ClassNotFoundException: com.fasterxml.jackson.databind.Module`

It seems like the jackson-databind dependency isn't being imported properly, which is likely causing the issue. Even though the version is forced, the dependency itself doesn't get added unless it's explicitly declared. That's why I still need the implementation lines in the dependencies block to ensure that these libraries are available in the project.

Reply:
I see. Let's discuss more on this. Approving the PR